et_feeder/et_feeder_node.h

```diff
@@ -49,7 +49,7 @@ class ETFeederNode {
 
   uint64_t id_;
   std::string name_;
-  bool is_cpu_op_;
+  uint32_t is_cpu_op_;
```

Comment:
Why is_cpu_op should be uint32_t?

Reply:
Was it because "inadequate support in protobuf for the oneof boolean value"?

Reply:
It seems that protobuf does not support oneof boolean values properly on some systems. That's why we are replacing it with uint32_t.

Reply:
Got it! Thank you.